Dockerfile

```diff
@@ -15,9 +15,4 @@ RUN conda create -n neurodsp python -q -y
 RUN ["/bin/bash", "-c", "source activate neurodsp && \
                         pip install -q jupyter && \
                         pip install -q -r requirements.txt && \
-                        python setup.py -q develop"]
-
-
-# If not in interactive mode, make the tutorials available over the exposed port
-COPY . /neurodsp
-CMD ["source activate neurodsp && jupyter notebook tutorials/ --ip=0.0.0.0 --port=5555 --no-browser --allow-root"]
+                        python setup.py -q develop"]
```

Comment:
When building the image using the Dockerfile, I ran into this error:
```
Traceback (most recent call last):
  File "setup.py", line 7, in <module>
    with open(os.path.join('neurodsp', 'version.py')) as version_file:
FileNotFoundError: [Errno 2] No such file or directory: 'neurodsp/version.py'
```
It looks like docker doesn't like the relative paths that are referenced in the new setup.py. The solution I found was to make to copy the version.py and README.rst to the container, along with with it's structure (ie neurodsp/version.py and README.rst at the base level). This required renaming the conda env. Here is the diff that allowed me to successfully build:
```
diff --git a/Dockerfile b/Dockerfile
index 650430c..61a0fbd 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -8,11 +8,16 @@ EXPOSE 5555
 RUN apt-get update -qq && apt-get upgrade -qq -y && conda update conda -q -y
 WORKDIR /neurodsp
 
+# This lets setup.py find correct files:
+RUN mkdir -p /neurodsp/neurodsp
+COPY neurodsp/version.py /neurodsp/neurodsp/version.py
+COPY README.rst /neurodsp/README.rst
+
 # Create the conda environment and install dependencies
 COPY requirements.txt /neurodsp/requirements.txt
 COPY setup.py /neurodsp/setup.py
-RUN conda create -n neurodsp python -q -y
-RUN ["/bin/bash", "-c", "source activate neurodsp && \
+RUN conda create -n env python -q -y
+RUN ["/bin/bash", "-c", "source activate env && \
                         pip install -q jupyter && \
                         pip install -q -r requirements.txt && \
                         python setup.py -q develop"]
```

Reply:
Thanks for digging in - as discussed, Dockerfile is now dropped!